### ISSUE TITLE
彭博士您好

### ISSUE THREAD

**Issue by lhaiqianyi** [ZhihaoPENG-CityU/Active-Transfer-Learning-Peng-et-al.-TCSVT19#1]
请问权重系数向量每次迭代以及最后的结果的累加和为什么不是1，这样的话没有满足限制条件的要求。

**Comment by ZhihaoPENG-CityU** [comment]
The original objective function (Eq. (8)) is difficult to solve because there are multiple constraints in the objective function. Therefore, we utilize ADMM to solve the optimization problem, making the value of "\mathbf{a}" in the range of 0 to 1, and the sum is close to 1. The illustration is given in Table Ⅶ.
Remark: This code is implemented in a typical non-deep approach rather than a deep neural network. In other words, we can achieve the deep learning version of active transfer learning by using the Softmax function to limit the value of "\mathbf{a}" in the range of 0 to 1 and its summation to 1.

**Comment by lhaiqianyi** [comment]
Thank you for your reply. I am very embarrassed to describe to you the problem I have now. Testing my bearing data set on your code, I visualized the results of the five categories and found that the selected source domain samples and target domain samples did not align marginal probability distributions. I have tried to change the data set, change the feature extraction method and adjust the parameters, but this problem has not been effectively solved. Do you have any unique ideas? Sorry to bother you. Looking forward to your reply.

**Comment by ZhihaoPENG-CityU** [comment]
Sorry for the late; I have been busy the past two weeks due to several upcoming conferences and reviewer works. In a nutshell, you should add a specific constraint loss function to align the marginal probability distributions. Please feel free to contact me, but it will be better to email me (zhihapeng3-c@my.cityu.edu.hk). @lhaiqianyi
